### PR TITLE
feat(news): implement news feed component

### PR DESCRIPTION
**Pull request by yhuard**

### CODE REVIEW

File: src/framework/molecules/panel/styles/panel.less [hunk 1]
```diff
@@ -0,0 +1,30 @@
+.f-panel {
```
Comment (kalinchernev) [inline src/framework/molecules/panel/styles/panel.less]: In the rest of the items I place `c-` namespace for component, but `f-` is also good, please check the rest and update.

File: src/framework/organisms/news/templates/news.html [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+---
+<section class="f-panel">
+  <header class="f-panel__header">
+    <h1 class="f-panel__header__title">News</h1>
+  </header>
+  <div class="f-panel__content">
+    <section class="f-news-feed">
+      <article class="f-news-feed__item">
+        <header>
+          <time class="f-news-feed__item__date" datetime="2016-06-17" pubdate="pubdate">17/06/2016</time>
```
Comment (yhuard) [inline src/framework/organisms/news/templates/news.html]: You're absolutely right! I'm changing it. Thanks

**Comment by yhuard** [comment]
We still have to find our way of working with namespaces but here's what I think:
1. A "level-of-implementation-specific" namespace could be helpful. Does the style come from the generic framework? The "site-specific-but-technology-agnostic" level (ie this style guide)? Or is it directly tied to the implementation?
2. Is it worth namespacing atoms, molecules, organisms? I mean, how often will you see an atom that is named exactly the same way as a molecule or an organism? Maybe this example is silly, but I can't imagine to find `a-button` and `m`button` or `o`button`... Right?

So yes, we need to namespace our classes, but as far as I know, only a "level-of-implementation-specific" namespace is helpful.

**Comment by yhuard** [comment]
I need this branch in master asap, that's why I used my admin privileges to merge it. I let the topic open for discussion though.

**Comment by kalinchernev** [comment]
Hi @yhuard of course, please go ahead.
My idea was basically coming from [this example](http://bradfrost.com/blog/post/css-architecture-for-design-systems/). 

> c- for UI components, such as .cn-c-card or .cn-c-header

I liked this idea since it's easy to re-use (copy+paste+modify a bit) components between style guides, yet avoiding conflicts with bootstrap for example. The brand namespace would definitely also be useful for larger projects, so I am really ok with it too. Thus, yes, I wouldn't use the a, m, o as the distinction is not always super-clear and sometimes takes a lot of efforts and discussions to pin down :)

**Pull request merged**